.github/workflows/build.yaml

```
@@ -472,34 +464,102 @@ jobs:
+
+          # Add splash if provided
+          if [[ -n "$SPLASH_IMAGE" && -f ".app/$SPLASH_IMAGE" ]]; then
+            vpk_args+=(--splashImage ".app/$SPLASH_IMAGE")
+          fi
+
+          "${vpk_args[@]}"
+
+      - name: Rename Velopack outputs
+        shell: bash
+        env:
+          PACK_ID: ${{ inputs.velopack_pack_id }}
```

actionlint (1.7.12)

[error] property "velopack_pack_id" is not defined in object type {channel: string; project: string}

(expression)